src/useTimeoutEffect/__tests__/ssr.ts

```diff
@@ -0,0 +1,13 @@
+import { renderHook } from '@testing-library/react-hooks/server';
+import { useTimeoutEffect } from '../..';
+
+describe('useTimeoutEffect', () => {
```

Comment:
i'd also add tests to ensure that callback is not invoked within SSR environment.

Reply:
Why should it not be invoked? Does `setTimeout` not work in SSR? I haven't worked with SSR yet, so I don't know about exactly all the details.

Reply:
Effects dont work in SSR, therefore callback of this hook should not be invoked in SSR. By naming hook as `*Effect` we must comply default behaviour